tasks/copy_to_glacier/README.md

```diff
@@ -242,6 +218,29 @@ The output of this lambda is a dictionary with a `granules` and `copied_to_glaci
 }
 ```
 
+## Configuration
+
+To configure a collection to enable ORCA, add the line `granuleRecoveryWorkflow: OrcaRecoveryWorkflow` to the collection configuration. It is also possible to exclude copying certain types of files to glacier bucket by adding the file type values to an `excludeFileTypes` variable.  An example of a collection configuration is given below:
```

Comment:
This is not what I had in mind here. For configuration, I would talk about `task_config` as part of the lambda configuration in the step function and how the `collection` key is needed. As well as showing how it should point to the collection configuration metadata that is part of CMA.

The information you have here, other than the link to the schema really should be in the website docs which I believe it is already.